.circleci/config.yml

```diff
@@ -15,7 +15,14 @@ jobs:
     <<: [ *defaults, *go-1_10 ]
     steps:
       - checkout
+      - restore_cache:
+          key: 'vendor'
       - run: 'make deps'
+      - save_cache:
```

Comment:
I looks like this doesn't cache the new data because the cache is immutable:
```
Skipping cache generation, cache already exists for key: vendor
Found one created at 2018-06-18 20:19:31 +0000 UTC
```

Reply:
I'd imagine its because nothing changed, but i'll make sure there isn't some `update_cache` type entry.